mbed_app.json

```diff
@@ -534,8 +528,9 @@
             "storage_tdb_internal.internal_size"        : "(32*2*1024)"
         },
         "NUCLEO_F767ZI": {
+            "target.features_add"                       : ["BOOTLOADER", "STORAGE"],
```

Comment:
"update-client.storage-locations"           : 1,

missing ?

Reply:
Already in place.